src/pages/Add2/Add2.jsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import FileBase from 'react-file-base64';
+/* import axios from 'axios'; */
+
+import './Add2.css';
+import './loader.css';
+import { useNavigate } from 'react-router-dom';
+
+import { addDoctor, finishDoctor } from '../../redux/doctors/adds';
+
+export default function Add2() {
+  /* Load Redux State */
+  const adds = useSelector((state) => state.add);
+
+  /* Setup Redux dispatch */
+  const dispatch = useDispatch();
+
+  const history = useNavigate();
+
+  // *********************************
+
+  const [listingData, setListingData] = useState('');
+  const [ansApi, setAnsApi] = useState('');
+  const [isLoader, setIsLoader] = useState(false);
+
+  let data64; // init variable
+
+  // console.log(listingData)
+
+  /* Put this Conditional for starting web app o refreshing
+   * to avoid errors because initial value is ""
+   * initial value:
+   * listingData = "" and listingData.selectedFile = undefined
+  */
+
+  if (listingData !== '') {
+    data64 = listingData.selectedFile.split(',');
+    // console.log(data64); //data64 = image converted to base64
+  }
+
+  // Input File
+  // Convert image to base 64
+  // set ansApi to false
+  const handleListing = ({ base64 }) => {
+    setListingData({ ...listingData, selectedFile: base64 });
+
+    setAnsApi('');
+  };
+
+  // Send image to ImgBB with AXIOS
+  const handleClick = () => {
+    // show loader hiden window
+    setIsLoader(!isLoader);
+
+    const data = new FormData();
+    data.append('image', data64[1]);
+    // data.append('name', 'prueba01')
+
+    // imgbb's personal Token
+    // https://es.imgbb.com/
+    const imgbbToken = '165bc83a2b0f87e5ddc8af943b7fcba4';
+    const APIurl = 'https://api.imgbb.com/1/upload?key=';
+
+    fetch(APIurl + imgbbToken, {
+      method: 'POST',
+      body: data,
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        // console.log(JSON.stringify(data));
```

Comment:
- [ ] Kindly, remove this old piece of commented out code in order to improve the repo's cleanliness. 👍